playbooks/keepalived.yaml

```diff
@@ -47,9 +47,35 @@
       vars:
         keepalived_role: MASTER
         kp_opposite_ip: "{{ hostvars[groups['keepalived-slave'][0]].ansible_host }}"
-        notify_down_action: ""
+        notify_down_action: "/etc/keepalived/master_notify.sh"
         local_interface: "{{ local_default_interface.stdout }}"
 
+    # TODO(bzhaoopenstack): For forbid pollute online ENV, we use a test account
+    # to test post issue and change the github app webhook.
+    - name: Copy necessary scripts
+      copy: src={{ item.src }} dest={{ item.dest }} mode=0755
+      with_items:
+        - { src: "{{ labkeeper_config_git_dest }}/keepalived/master_notify.sh.j2", dest: "/etc/keepalived/master_notify.sh" }
```

Comment:
same.

Reply:
we cannot use copy to template jinja2 file